tests/e2e/Services/Account/AccountCustomServerTest.php

```diff
@@ -6,35 +6,154 @@
 use Tests\E2E\Scopes\ProjectCustom;
 use Tests\E2E\Scopes\Scope;
 use Tests\E2E\Scopes\SideServer;
+use Utopia\Database\Validator\Datetime as DatetimeValidator;
 use Utopia\Database\Helpers\ID;
 
 class AccountCustomServerTest extends Scope
 {
     use ProjectCustom;
     use SideServer;
 
-    public function testCreateAccount(): array
+
+    public function testCreateAnonymousAccount()
```

Comment:
what about the email and password route? Is that not accessible to the server?

Reply:
Added a test for it :)